v2/attrs.go

```diff
@@ -51,9 +51,9 @@ type attrsKey struct{}
 //
 // Usage:
 //
-//	ctx := log.WithCtx(ctx, "height", 1234)
+//	unusedCtx := log.WithCtx(unusedCtx, "height", 1234)
 //	...
-//	log.Info(ctx, "Height processed") // Will contain attribute: height=1234
+//	log.Info(unusedCtx, "Height processed") // Will contain attribute: height=1234
 func WithCtx(ctx context.Context, attrs ...any) context.Context {
```

Comment:
unrelated, just noting here, that passing an odd number of attrs is possible here, not sure if we'd want to prevent that

Reply:
this is the same for anything logging related. I dont think loggers should ever prevent anything  - we never want loggers to cause panics and we dont want to handle errors from loggers. So things like this will just result in "BADKEY" being printed as the value (since there will be a key with no value)